examples/builder.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+from tierkreis.cli.run_workflow import run_workflow
+
+from tierkreis.controller.data.graph import GraphData
+from tierkreis.controller.data.graph_builder import AutoGraphBuilder, GraphBuilder
+from tierkreis import Labels
+
+
+def loop_body() -> GraphData:
```

Comment:
I'm pretty unconvinced by this approach, while it's less verbose it seems like it would be very awkward to make changes as adding new nodes in between existing nodes will mean updating every index. It's also very error prone to work this way as you are very vulnerable to off by one errors.

I guess if you have some degree of validation you might be able to avoid some issues, but I'm not sure this is very workable as an approach.